operator/templates/cassandra-exporter-config-yml.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cassandra-exporter-config-yml
+data:
+  cassandra-exporter-config.yml: |
+    host: localhost:{{ .Params.JMX_PORT }}
+    ssl: False
+    user:
+    password:
+    listenAddress: 0.0.0.0
+    listenPort: {{ .Params.METRICS_PORT }}
+    blacklist:
+       # To profile the duration of jmx call you can start the program with the following options
+       # > java -Dorg.slf4j.simpleLogger.defaultLogLevel=trace -jar cassandra_exporter.jar config.yml --oneshot
+       #
+       # To get intuition of what is done by cassandra when something is called you can look in cassandra
+       # https://github.com/apache/cassandra/tree/trunk/src/java/org/apache/cassandra/metrics
+       # Please avoid to scrape frequently those calls that are iterating over all sstables
+
+       # Unaccessible metrics (not enough privilege)
+       - java:lang:memorypool:.*usagethreshold.*
+
+       # Leaf attributes not interesting for us but that are presents in many path
```

Comment:
Was this copy paste from Criteo's config? Some of these attributes _do_ seem interesting... but maybe it exposes too many metrics.

Reply:
not actually recommended but example :   https://github.com/criteo/cassandra_exporter#config-file-example

Reply:
@samvantran keep in mind that this is the blacklist.